docs/design.md

```diff
@@ -91,6 +91,13 @@ When deciding between different design options, we follow the following principl
 * MicroShift uses as few configuration options as possible. Where it provides configuration options, they are intuitive and have sensible defaults respectively are auto-configured.
 * MicroShift is preferably configured through config files, but allows overriding of parameters via environment variables (for use in containers, systemd) and command line flags (for ad-hoc use).
 * MicroShift can use both user-local and system-wide configuration.
+* Components built into the `microshift` binary are configured by modifying `microshift.conf` and restarting the service, instead of through Kubernetes APIs. This ensures that
```

Comment:
`microshift.conf` --> `/etc/microshift/config.yaml` ?

Reply:
Yes, I keep making that mistake.